indico/modules/events/registration/models/registrations.py

```diff
@@ -226,6 +249,17 @@ class Registration(db.Model):
             lazy=True
         )
     )
+    #: The registration tags assigned to this registration
+    tags = db.relationship(
+        'RegistrationTag',
+        secondary=registrations_tags_table,
+        cascade='all, delete',
```

Comment:
I think this may delete a tag when hard-deleting (which we usually don't do) the corresponding registration!
https://docs.sqlalchemy.org/en/14/orm/cascades.html#delete

I don't think we need a `cascade` setting here at all